test/CMakeLists.txt

```diff
@@ -2,13 +2,11 @@ cmake_minimum_required(VERSION 2.8)
 
 project(azure_storage_test)
 
-set(AZURE_STORAGE_TEST_HEADER
+set(AZURE_STORAGE_TEST_SOURCES
```

Comment:
>AZURE_STORAGE_TEST_SOURCES [](start = 4, length = 26)

If you think it is necessary to merge the source/headers into single value, please make sure the name is also changed.

Reply:
This is just a variable used in current `CMakeLists.txt` file, without any effects outside this file. I just don't think it's necessary to split headers/sources.